app/lib/package/backend.dart

```diff
@@ -123,11 +123,21 @@ class PackageBackend {
         .map((p) => p.name!);
   }
 
-  /// Retrieves package versions ordered by their latest version date.
+  /// Retrieves package versions ordered by their published date descending.
   Future<List<PackageVersion>> latestPackageVersions(
-      {int? offset, int? limit}) async {
-    final pkgPage = await latestPackages(offset: offset, limit: limit);
-    return lookupLatestVersions(pkgPage.packages);
```

Comment:
Do we use `lookupLatestVersions` anywhere else? Otherwise it might be dead code.